fetchy/utilities.ts

```diff
@@ -1,2 +1,8 @@
-export const convertToNumber = (s: string): number =>
-  Number(s.replace(/[%,]/g, ""));
+export const convertToNumber = (s: string): number => {
+  const result = Number(s.replace(/[%,]/g, ""));
+  console.log(result);
+  if (isNaN(result) || result === null || result === undefined) {
+    throw new Error(`Could not convert string ${s} to a number.`);
+  }
+  return result;
```

Comment:
I don't think `Number(x)` can return `null` or `undefined` so those checks are probably redundant. Also Number("") returns `0` (is that what we want??)

Soooo I think we might want something more like:

```suggestion
  const stripped = s.replace(/[%,]/g, ""); // strip known non-digit characters
  const result = Number(stripped || NaN); // convert to number or NaN by default
  if (Number.isNaN(result)) {
    throw new Error(`Could not convert string ${s} to a number.`);
  }
  return result;
```